source/includes/read/change-streams.php

```diff
@@ -0,0 +1,47 @@
+<?php
+require 'vendor/autoload.php'; 
+
+$uri = getenv('MONGODB_URI') ?: throw new RuntimeException('Set the MONGODB_URI variable to your Atlas URI that connects to the sample dataset');
+$client = new MongoDB\Client($uri);
+
+// start-db-coll
+$collection = $client->sample_restaurants->restaurants;
+// end-db-coll
+
+// Monitors and prints changes to the "restaurants" collection
```

Comment:
Are these comments intended for the readers, or just the doc authors. Asking as it appears outside of the ``start`` and ``end`` blocks used for including code in the rendered docs.

Reply:
These comments are for AI ingestion to clarify what the code's doing. We try not to make them visible on the page if possible